wait_for/__init__.py

```diff
@@ -233,9 +264,11 @@ def wait_for(func, func_args=[], func_kwargs={}, logger=None, **kwargs):
     if not silent_fail:
         logger.error(logger_fmt, logger_dict)
         logger.error('The last result of the call was: %(result)r', {'result': out})
+
         if raise_original and exc:
             raise exc
-        raise TimedOutError(timeout_msg)
+        else:
```

Comment:
This slight change is in and of itself a great addition to the behavior on time out, thanks! This will allow for better introspection into the fail condition and behavior of func() during the wait loop.